TiDB_FAQ.md

```diff
@@ -0,0 +1,139 @@
+
+## What does Placement Driver (PD) do?
+
+Placement Driver (PD) works as the cluster manager of TiDB. It manages the TiKV metadata and makes decisions for data placement and load balancing. PD periodically checks replication constraints to balance load and data automatically.
+
+## Is it easy to use TiDB?
+
+Yes, it is. When all the required services are started, you can use TiDB as easily as a MySQL server.  You can replace MySQL with TiDB to power your applications without changing a single line of code in most cases. You can also manage TiDB using the popular MySQL management tools.
+
+## When to use TiDB?
+
+TiDB is at your service if your applications require any of the following:
++ Horizontal scalability
++ Business continuity 
++ Strong consistency
++ Support for distributed ACID transactions
+
+## When not to use TiDB?
+
+TiDB is not a good choice if the number of the rows in your database table is less than 100GB and there is no requirement for high availability, strong consistency and cross-datacenter replication.
+
+## How does TiDB manage user account?
+
+TiDB follows MySQL user account management system. You can create user accounts and authorize them. 
+
+You can use MySQL grammar to create user accounts. For example, you can create a user account by using the following statement:
+
+CREATE USER 'test'@'localhost' identified by '123';
+
+The user name of this account is "test"; the password is “123" and this user can login from localhost only.
+
+You can also use MySQL grammar to authorize this user. For example, you can grant the read privilege to the "test" user by using the following statement:
+
+GRANT SELECT ON \*.\* TO  'test'@'localhost';
+
+Note: There are following differences between TiDB and MySQL in user account creating and authorizing:
+
++ You can use the "user@hostname" grammar to create accounts. In this grammar, “hostname” supports exact match like “192.168.199.1” and full match like “%”, but doesn’t support prefix match like “192.168.%”.
+
++ To be compatible with the existing MySQL businesses, TiDB supports user authorization but it only supports the authorization grammar and recording the authorization in the system table. TiDB checks the authorization for the DropTable statement only. It does not check the authorization for other statements.
+
+## How does TiDB scale?
+
+As your business grows, your database might face the following three bottlenecks:
+
++ Lack of storage resources which means that the disk space is not enough.
+
++ Lack of computing resources such as high CPU occupancy.
+
++ Not enough throughputs.
```

Comment:
"No" can be used as a noun, adj. and adv. While "not" is mostly used as an adv. When "No" is used as an adv., it is usually used in two situations:
1.The antonym of "Yes" to answer a question.
Are you going to Shanghai today?
No, I am not.
2. To embellish a comparative form: There is no better choice than fleeing the country.
However, when it comes to embellish word like "a", "any", "much", "many", "enough", etc,. we can only use "not".